SpatialGDK/Source/SpatialGDK/Public/Tests/SpatialView/TargetView.h

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/EntityView.h"
+#include "SpatialView/MessagesToSend.h"
+#include "SpatialView/OpList/EntityComponentOpList.h"
+
+namespace SpatialGDK
+{
+/**
+ * Generates OpLists to describe changes to a View.
+ * No further changes can be made after calling Disconnect.
+ * Functions that add things to the view will add the entity if it does not already exist.
+ * Functions that remove or modify things require that thing to have previously existed.
+ * Functions that add command requests and responses have no entity prerequisites.
+ */
+class FTargetView
```

Comment:
Add a comment, hinting at how it should be used when simulating discussions between workers, because unlike the EntityOpBuilder, it is not transient, and should be tied to another view coordinator so their view does not desynchronize.

Reply:
Maybe the TestWorker thing should actually have a built in TargetView

Reply:
Yeah, could be.